src/MessageHandler/GithubImportCommandHandler.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace App\MessageHandler;
+
+use App\Message\GithubImportCommand;
+use App\Message\ImportCommandCreator;
+use App\Services\GithubService;
+use Composer\Util\Git;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class GithubImportCommandHandler implements MessageHandlerInterface {
+    private GithubService $githubService;
+
+    public function __construct(GithubService $githubService) {
+        $this->githubService = $githubService;
+    }
+    public function __invoke(GithubImportCommand $command) {
+        $this->githubService->importToDb($command->getOrgName());

```

Comment:
In such case importToDb is not needed any more you can move whole code from that method into handler directly

Reply:
Solved